src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -147,29 +187,36 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
             <Textfield
               label={strings.SUBSET_COUNT}
               id='subsetCount'
-              onChange={(value) => onChange('subsetCount', value)}
-              type='text'
+              onChange={(value) => onChangeSubsetCount(value as number)}
+              type='number'
+              min={0}
+              disabledCharacters={['.', '-']}
```

Comment:
Will this do the right thing if you're using a locale that uses comma as the decimal separator?

Reply:
oops good catch, this list had a comma not sure why I removed it.. will bring it back